tests/secureserver.pl

```diff
@@ -230,8 +230,8 @@ sub exit_signal_handler {
     $tstunnel_windows = 1;
 
     # replace Cygwin and MinGW drives within paths
-    $capath =~ s/^(\/cygdrive)?\/(\w)\//$2\:\//;
-    $certfile =~ s/^(\/cygdrive)?\/(\w)\//$2\:\//;
+    chomp($capath = `cygpath -m -a $capath`);
+    chomp($certfile = `cygpath -m -a $certfile`);
```

Comment:
I think these do not need to be changed, because this a conversion from Cygwin and msys to Windows style paths. The cygdrive part is actually optional here.